src/cluster-ev-builder.cc

```diff
@@ -2142,6 +2142,15 @@ void EVBuilder::analyzeTransferMatrix(Vertex const& v,
     std::vector<std::complex<double>> ev;
     std::vector<double> V;
     ardns.real_nonsymm(N, 4, 100, 0.0, N * 10, ev, V);
+    
+    //sort
+    std::sort(ev.begin(), ev.end(), [](std::complex<double> a, std::complex<double> b) {
```

Comment:
pass by reference: `const std::complex<double>& a, ...`
since we are using `c++14`, if you will you can also use `auto`

```
[]( const auto& a, const auto& b){
```